### PR TITLE
Security fixes (PRNG->RNG random) + JSON RPC finally moved from Ether…

### LINKED ISSUE #52: Private key generation from insecure random mnemonic phrases.
Ethereum Harmony generates private keys using weak PRNGs functions (Java's java.util.random and Javascript's Math.random)
Funds sent to wallets generated using Ethereum Harmony could be at risk.

### PR DESCRIPTION
**Pull request by zilm13**
@mkalinin please, check
- All random generators changed from PRNG to RNG
- Minor frontend security fixes
- JSON RPC and its dependencies finally moved from EthereumJ to Harmony (one of it may be insecure so it was good time to do)
- Note added to popup "Mnemonic address creation" that mnemonic phrase is less secure than private key.
Should resolve #52

### CODE REVIEW

File: src/main/resources/static/pages/popups/phraseAddress.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="modal-content">
             <div class="modal-header">
                 <h4 class="modal-title">Generate address with mnemonic phrase</h4>
-                <span>Be sure to remember phrase and write it on paper. Private key will not be kept on server.</span>
+                <span>Mnemonic phrase is less secure than private key, but it's easy to remember. Phrase/private key will not be kept on server.</span>
```
Comment (ligi) [inline src/main/resources/static/pages/popups/phraseAddress.html]: why is a Mnemonic phrase less secure than a private key?

Comment (zilm13) [reply src/main/resources/static/pages/popups/phraseAddress.html]: It's about 128 bits of entropy in it while private key is 256 bit.

Comment (ligi) [reply src/main/resources/static/pages/popups/phraseAddress.html]: is this for 12 word mnemonics?

Comment (zilm13) [reply src/main/resources/static/pages/popups/phraseAddress.html]: @ligi we use 5 words + 1 pseudo-word
12 words is less than 256 bit too but has more entropy than our approach
it's a compromise between ability to remember and strength
the thing you couldn't remember you will write on the paper or store on disk, both of which are security threats too

**Pull request merged**